source/Trashbot.js

```diff
@@ -36,9 +37,10 @@ export default class Trashbot extends Pixi.Sprite {
     }
 
     onCollision(collidedWith) {
+        var finalPosition = this.position
```

Comment:
This is my same style for writing functions/methods. I always start with simplifying the signature of complex variable names into simpler names because I feel it lends to legibility. Good stuff.